content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -85,6 +85,14 @@ The current version of the application (e.g., a semantic version, revision hash,
 
 One of the [recommended labels](/docs/concepts/overview/working-with-objects/common-labels/#labels).
 
+### cluster-autoscaler.kubernetes.io/safe-to-evict
+
+Example: `cluster-autoscaler.kubernetes.io/safe-to-evict: true`
+
+Used on: Pod
+
+When this annotation is set to `true`, Pods can be evicted even if they don't meet other requirements for it.
```

Comment:
```suggestion
When this annotation is set to `true`, the cluster autoscaler is allowed to evict a Pod
even if other rules would normally prevent that.
```

What about if you set it to `false`? We could also explain that if you don't set this annotation then the cluster autoscaler follows its overall behavior.

Reply:
Done